README.md

```diff
@@ -1,15 +1,25 @@
 # OpenMLS [![OpenMLS Chat][chat-image]][chat-link]
 
-[![Tests & Checks](https://github.com/openmls/openmls/actions/workflows/tests.yml/badge.svg)](https://github.com/openmls/openmls/actions/workflows/tests.yml)
-[![ARM64 Build Status](https://cloud.drone.io/api/badges/openmls/openmls/status.svg?ref=refs/heads/main)](https://cloud.drone.io/openmls/openmls)
-[![Deploy Docs](https://github.com/openmls/openmls/workflows/Deploy%20Docs/badge.svg)][docs-main-link]
-[![codecov](https://codecov.io/gh/openmls/openmls/branch/main/graph/badge.svg?token=5SDRDRTZI0)](https://codecov.io/gh/openmls/openmls)
+[![Tests & Checks][gh-tests-image]](https://github.com/openmls/openmls/actions/workflows/tests.yml)
+[![ARM64 Build Status][drone-image]](https://cloud.drone.io/openmls/openmls)
+[![Deploy Docs][gh-deploy-docs-image]][docs-main-link]
```

Comment:
Not sure if we really need this one. We have a ton of jobs but this one is probably not a very interesting one at this point.